contracts/src/base/registry/facets/checker/EntitlementChecker.sol

```diff
@@ -163,32 +163,27 @@ contract EntitlementChecker is IEntitlementChecker, Facet {
   /**
    * @notice Get the nodes registered by an operator
    * @param operator The address of the operator
-   * @return An array of node addresses
+   * @return nodes An array of node addresses
    */
   function getNodesByOperator(
     address operator
-  ) external view returns (address[] memory) {
+  ) external view returns (address[] memory nodes) {
     EntitlementCheckerStorage.Layout storage layout = EntitlementCheckerStorage
       .layout();
     uint256 totalNodeCount = layout.nodes.length();
-    uint256 nodeCount = 0;
-    for (uint256 i = 0; i < totalNodeCount; i++) {
+    nodes = new address[](totalNodeCount);
+    uint256 nodeCount;
+    for (uint256 i; i < totalNodeCount; ++i) {
```

Comment:
no need to use ++i vs i++ since newer compiler optimizes for this, can leave but don't want to go down the rabbit hole of over optimizing small things.

Reply:
Agree.